pegasus/sites.v3/hourofcode.com/i18n/en.yml

```diff
@@ -31,6 +31,9 @@
   oceans_landing_cs_for_good: 'CS For Good'
   oceans_landing_cs_for_good_desc: 'Resources to help students understand the role computer science could play in creating a more equitable world.'
 
+  social_hoc2019_oceans_title: 'AI for Oceans #CSForGood'
+  social_hoc2019_oceans_desc: "Learn about AI, machine learning, training data, and bias, while exploring ethical issues and how AI can be used to address world problems. Computer science is about so much more than coding! Enjoy Code.org's first step in a new journey to teach more about AI throughout our curriculum."
```

Comment:
Oh I'm curious if this string will still fit everywhere okay?  It seems a bit long?

Reply:
That's the copy Kirsten gave me. If it needs to be shortened we can do it in a follow-up?